prow/jobs/jinja/postsubmits/runtime_release.jinja2

```diff
@@ -0,0 +1,31 @@
+  aws-controllers-k8s/runtime:
+  - name: runtime-docs-release
+    decorate: true
+    extra_refs:
+    - org: aws-controllers-k8s
+      repo: community
+      base_ref: main
+      workdir: true
+    {% for service in aws_services %}
+    - org: aws-controllers-k8s
+      repo: {{ service }}-controller
+      base_ref: main
+    {% endfor %}
+    spec:
+      serviceAccountName: post-submit-service-account
+      containers:
+        - image: public.ecr.aws/m5q3e4b2/prow:prow-docs-0.0.3
+          env:
+          - name: GITHUB_TOKEN
+            valueFrom:
+              secretKeyRef:
+                name: prowjob-github-token
+                key: token
+          - name: GITHUB_ACTOR
+            value: "ack-bot"
+          - name: GITHUB_EMAIL_ID
+            value: "82905295"
+          command: ["/docs/build-docs.sh"]
+    run_if_changed: "apis/core/.*"
+    branches:
+    - main
```

Comment:
Where is this file being included in a generated ProwJob YAML?

Reply:
This is being included as a postsubmit job. Our script knows this because it's under the `jinja/postsubmits/` directory. Your second comment is where the generator put it in the generated `jobs.yaml`

Reply:
Gotcha, thanks for the explanation :)